hardhat/tasks/sips/args/sipArgs.js

```diff
@@ -981,6 +981,51 @@ const getArgsSip0074 = async (hre) => {
     return { args, governor: "GovernorOwner" };
 };
 
+const getArgsSip0076 = async (hre) => {
+    // Electron release
+    const {
+        ethers,
+        deployments: { get },
+    } = hre;
+    const abiCoder = new ethers.utils.AbiCoder();
+
+    const adoptionFundDeployment = await get("AdoptionFund");
+    const devFundDeployment = await get("DevelopmentFund");
+    const sovTokenDeployment = await get("SOV");
+    const multisigDeployment = await get("MultiSigWallet");
```

Comment:
Please, replace "MultiSigWallet" to the right deployment, the new Gnosis Wallet: 0xdd8e07a57560ada0a2d84a96c457a5e6ddd488b7
The one we are pointing here is the old exchequer which is meant to be only the future Guardian:
0x924f5ad34698Fd20c90Fe5D5A8A0abd3b42dc711